### ISSUE TITLE
Bug: Dark Mode Issue - Table Content Not Easily Visible

### ISSUE THREAD

**Issue by crespire** [TheOdinProject/theodinproject#2644]
![image](https://user-images.githubusercontent.com/36272822/153899705-206d9754-dcb2-4e7b-8e2e-78ef9844999e.png)

Using Odin Dark Mode, this table in the [Rails Active Record Associations lesson](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/active-record-associations#source) is not readable.

**Comment by swaubhik** [comment]
I would like to work with this issue.

**Comment by ChargrilledChook** [comment]
Thanks heaps for the report @crespire. The site is currently migrating all its CSS to use Tailwind, so the fix for this should be covered by the work being done there. We'll keep this open to track it and make sure a fix gets in.

**Comment by KevinMulhern** [comment]
This has been fixed with the [new dark mode](https://github.com/TheOdinProject/top-meta/issues/235)
<img width="760" alt="Screenshot 2022-10-13 at 19 25 24" src="https://user-images.githubusercontent.com/7963776/195676661-c3a906f6-fbd1-49aa-8063-4c830ea47662.png">
